pkg/eniconfig/eniconfig_test.go

```diff
@@ -68,6 +68,33 @@ func updateNodeAnnotation(hdlr sdk.Handler, nodeName string, configName string,
 	hdlr.Handle(nil, event)
 }
 
+func updateNodeLabel(hdlr sdk.Handler, nodeName string, configName string, toDelete bool) {
```

Comment:
This function is almost same with `updateNodeAnnotation()`. It is better to combine them?

Reply:
Can we extract the strictly duplicate code.  I think two methods is fine, if we remove the code that is the same toa  `createEvent` function.